vue/components/ui/input-platforme/input-platforme.vue

```diff
@@ -1,22 +1,52 @@
 <template>
-    <input
-        type="text"
-        class="input-platforme"
-        v-bind:value="value"
-        v-bind:placeholder="placeholder"
-        ref="input"
-        v-on:input="$emit('update:value', $event.target.value)"
-        v-on:focus="focused()"
-        v-on:blur="blurred()"
-    />
+    <div class="input-platforme">
+        <input
+            type="text"
+            class="input"
+            v-bind:value="value"
+            v-bind:placeholder="placeholder"
+            v-bind:disabled="disabled"
+            ref="input"
+            v-on:input="$emit('update:value', $event.target.value)"
+            v-on:focus="focused()"
+            v-on:blur="blurred()"

```

Comment:
Since you're here please adapt this to be more similar to other components: `onX` instead of `x()`